src/lib/sw_engine/tvgSwFill.cpp

```diff
@@ -210,12 +209,12 @@ void fillFetchRadial(const SwFill* fill, uint32_t* dst, uint32_t y, uint32_t x,
     //Rotation
     auto rx = (x + 0.5f - fill->radial.cx) * fill->sy;
     auto ry = (y + 0.5f - fill->radial.cy) * fill->sx;
-    auto inv2a = fill->radial.inv2a;
     auto rxy = rx * rx + ry * ry;
     auto rxryPlus = 2 * rx;
-    auto det = (-4 * fill->radial.a * -rxy) * inv2a;
-    auto detDelta = (4 * fill->radial.a * (rxryPlus + 1.0f)) * inv2a;
-    auto detDelta2 = (4 * fill->radial.a * 2.0f) * inv2a;
+    auto inva = 1.0 / fill->radial.a;
```

Comment:
Apart from the logic changes,

if we keep the inva in the data structure (as it does), we can avoid dividing operation per pixels.

Reply:
I restored it in the struct, but I used inva instead of inv2a. Am I missing something? do we need inv2a?

Reply:
@mgrudzinska neither i have no idea. what brought you to change to this code. since you introduced new code, i wish you can be sure by yourself first... :)